extending/provider/implementation.rst

```diff
@@ -111,35 +63,39 @@ The following example implementation of ``refresh()`` updates an EBS volume in A
          setState(volume.stateAsString());
          setVolumeType(volume.volumeTypeAsString());
 
-         Ec2Client client = createClient(Ec2Client.class);
-
          DescribeVolumeAttributeResponse responseAutoEnableIo = client.describeVolumeAttribute(
-             r -> r.volumeId(getId())
-                 .attribute(VolumeAttributeName.AUTO_ENABLE_IO)
+             r -> r.volumeId(getId()).attribute(VolumeAttributeName.AUTO_ENABLE_IO)
          );
 
          setAutoEnableIo(responseAutoEnableIo.autoEnableIO().value());
+
+         return true;
      }
 
-**create(GyroUI ui, State state)**
+Implementing create
+===================
+
+The ``create(GyroUI ui, State state)`` method is called to create a resource. Gyro will call this method for every resource
+it determines needs to be created.
```

Comment:
I would briefly add what Gyro uses to make the determination to call ``create(...)`` versus ``update(...)``.